redux-form/lib/reduxForm.d.ts

```diff
@@ -147,14 +126,13 @@ export interface Config<FormData extends DataShape, P, S> {
     propNamespace?: string;
 
     /**
-     * The use of this property is highly discouraged, but if you absolutely
-     * need to mount your redux-form reducer at somewhere other than form in
-     * your Redux state, you will need to specify the key you mounted it under
-     * with this property. Defaults to 'form'.
-     *
-     * See Alternate Mount Point Example for more details.
+     * If true, implements `shouldComponentUpdate` and shallowly compares _only_
+     * the Redux-connected props that are needed to manage the form state. This
+     * prevents unnecessary updates, but assumes the component is a "pure"
+     * component and does not rely on any input or state other than its props and
+     * the selected Redux store's state. Defaults to true.
      */
-    reduxMountPoint?: string;
```

Comment:
Looks like this is not part of public API anymore: http://redux-form.com/6.5.0/docs/api/ReduxForm.md/